presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -382,3 +386,88 @@ node.
 
 You should ensure that the keytab files have the correct permissions on every
 node after distributing them.
+
+File Based Authorization
+========================
+
+The config file is specified in JSON and is made up of three main sections, each of
+which is a list of rules which are matched first to last. The privileges from the
+first rule that matches are used. All regexes default to ".*", if not specified.
+
+Schema Rules
+------------
+
+These rules govern who is considered an owner of a schema.
+
+* ``user`` (optional): regex to match against user name.
+
+* ``schema`` (optional): regex to match against schema name.
+
+* ``owner`` (required): boolean indicating ownership.
+
+Table Rules
+-----------
+
+These rules govern the privileges granted on specific tables.
+
+* ``user`` (optional): regex to match against user name.
+
+* ``schema`` (optional): regex to match against schema name.
+
+* ``table`` (optional): regex to match against table name.
+
+* ``privileges`` (required): zero or more of ``SELECT``, ``INSERT``,
+  ``DELETE``, ``OWNERSHIP``, ``GRANT_SELECT``.
+
+Session Property Rules
+----------------------
+
+These rules govern who may set session properties.
+
+* ``user`` (optional): regex to match against user name.
+
+* ``sessionProperty`` (optional): regex to match against session property name.
+
+* ``allowed`` (required): boolean indicating whether this session property may be set.
+
+
+See below for an example.
+
+.. code-block:: json
+
+    {
+      "schemas": [
+        {
+          "user": "admin",
+          "schema": ".*",
```

Comment:
I think it's better to leave it to show that regexes can be used, in case the user doesn't read the text above